tests/aggregation/tests.py

```diff
@@ -756,7 +756,8 @@ def test_dates_with_aggregation(self):
                 "datetime.date(1995, 1, 1)",
                 "datetime.date(2007, 1, 1)",
                 "datetime.date(2008, 1, 1)"
-            ]
+            ],
+            repr,
```

Comment:
I think we should update assertions, instead of passing `repr`.

Reply:
I changed the assertion to `assertSequenceEqual`